modules/support-get-cluster-id.adoc

```diff
@@ -5,21 +5,31 @@
 [id="support-get-cluster-id_{context}"]
 = Obtaining your cluster ID
 
-When providing information to Red Hat Support, it is helpful to provide the unique identifier for your cluster. You can obtain your cluster ID by using the OpenShift CLI (`oc`) or by using the {product-title} web console.
+When providing information to Red Hat Support, it is helpful to provide the unique identifier for your cluster. You can have your cluster ID autofilled by using the {product-title} web console. You can also manually obtain your cluster ID by using the web console or the OpenShift CLI (`oc`).
 
 .Prerequisites
 
 * Access to the cluster as a user with the `cluster-admin` role.
-* The {product-title} CLI (`oc`) installed.
+* Access to the web console or the OpenShift CLI (`oc`) installed.
 
 .Procedure
+ifdef::openshift-enterprise,openshift-dedicated[]
+* To open a support case and have your cluster ID autofilled using the web console:
+.. From the toolbar, navigate to *(?) Help* -> *Open Support Case*.
+.. The 'Cluster ID' value is autofilled.
+endif::[]
+ifdef::openshift-origin[]
+* To open a bug and have your cluster ID autofilled using the web console:
+.. From the toolbar, navigate to *(?) Help* -> *Report Bug*.
+.. The 'Cluster ID' value is autofilled.
```

Comment:
^^ Same as comment above. Not sure clusterID is autofilled in BZ.

Reply:
It's put into the bugzilla environment, field. What Bz did you test with?

Reply:
Got it. I'm going to update to say that it is autofilled after you click `Submit bug`. Thanks!